workflows/sample_workflow.ts

```diff
@@ -0,0 +1,52 @@
+import { DefineWorkflow, Schema } from "deno-slack-sdk/mod.ts";
+import { SampleFunctionDefinition } from "../functions/sample_function.ts";
+
+const SampleWorkflow = DefineWorkflow({
+  callback_id: "sample_workflow",
+  title: "Sample workflow",
+  description: "A sample workflow",
+  input_parameters: {
+    properties: {
+      interactivity: {
+        type: Schema.slack.types.interactivity,
+      },
+      channel: {
+        type: Schema.slack.types.channel_id,
+      },
+    },
+    required: ["interactivity"],
+  },
+});
+
+const inputForm = SampleWorkflow.addStep(
+  Schema.slack.functions.OpenForm,
+  {
+    title: "Send message to channel",
+    interactivity: SampleWorkflow.inputs.interactivity,
+    submit_label: "Send message",
+    fields: {
+      elements: [{
+        name: "channel",
+        title: "Channel to send message to",
+        type: Schema.slack.types.channel_id,
+        default: SampleWorkflow.inputs.channel,
+      }, {
+        name: "message",
+        title: "Message",
+        type: Schema.types.string,
```

Comment:
We could add `long: true` as a property on this field element, which I think results in a little nicer form experience for this use-case (even though it's quite simple).

![image](https://user-images.githubusercontent.com/367275/187712658-69980d68-ff70-47f2-8ff4-7e90b37b7ca1.png)

Reply:
ooo i didn't know that! Nice

Reply:
I also had no idea this existed! I'll make the adjustment.